apps/backend/src/users/users.service.ts

```diff
@@ -37,6 +38,40 @@ export class UsersService {
     return await this.prisma.user.findMany({ omit: { weight: true } });
   }
 
+  async calBac(authSchId: string): Promise<UserBac> {
```

Comment:
this is quite an inmportant method but it's hard to say what it does just from its name. Either rename it or add some comments